.github/workflows/create-test-plan.py

```diff
@@ -98,49 +99,51 @@ class JobSpec:
     clang_cl: bool = False
     gdk: bool = False
     vita_gles: Optional[VitaGLES] = None
+    harmony_arch: Optional[str] = None
 
 
 JOB_SPECS = {
-    "msys2-mingw32": JobSpec(name="Windows (msys2, mingw32)",               os=JobOs.WindowsLatest,     platform=SdlPlatform.Msys2,       artifact="SDL-mingw32",            msys2_platform=Msys2Platform.Mingw32, ),
```

Comment:
Please undo these comments, and just make sure to add `[sdl-ci-filter harmony]` in the message of your last commit. This script will then only build haiku and openharmony.